src/PiWeb.Volume.Compare/PiWeb.Volume.Compare.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net8.0-windows10.0.22621</TargetFramework>
```

Comment:
can we do this without targeting windows? `<TargetFramework>net8.0</TargetFramework>`?

Reply:
that way we can use a linux runner

Reply:
Unfortunately not, that's a WPF project.